_studio/shared/asc/include/asc_avx2_impl.h

```diff
@@ -22,6 +22,28 @@
 #define _ASC_AVX2_IMPL_H_
 
 #include "asc_common_impl.h"
+// Load 0..7 floats to YMM register from memory
+// NOTE: elements of YMM are permuted [ 4 2 - 1 ]
+static inline __m256 LoadPartialYmm(
```

Comment:
These are just declarations. I think 'static inline' will be just ignored. Do we really need them here?

Reply:
We do not need them, I am cleaning that out.